### PR TITLE
Update directory diagram for extended administration modules

### PR DESCRIPTION
**Pull request by sushmangupta**
This is related to issue : #519

### CODE REVIEW

File: guides/plugins/plugins/administration/add-new-tab.md [hunk 1]
```diff
@@ -223,13 +223,13 @@ As shown in the previous example, your custom component is expected to be in a d
...
-├── page
-│   └── sw-product-detail
+├── src
+│   └──page
 │       └── sw-product-detail
 │           ├── index.js
 │           └── sw-product-detail.html.twig
-├── view
-│   └── sw-product-detail-custom
+│       └── view
```
Comment (bojanrajh) [inline guides/plugins/plugins/administration/add-new-tab.md:231]: It [seems](https://github.com/shopware/docs/blob/d943dd224897e59fe4fdc05b28d44fa3a3b00924/guides/plugins/plugins/administration/add-new-tab.md#creating-your-new-component) like `view` needs to be on the same level as `page` - please, revert changes in indentation for `view`, and keep the indentation as it is for `sw-product-detail-custom` (as it was fixed correctly).

**Review by bojanrajh — CHANGES_REQUESTED** [review]

File: guides/plugins/plugins/administration/add-new-tab.md [hunk 2]
```diff
@@ -223,13 +223,13 @@ As shown in the previous example, your custom component is expected to be in a d
 
 ```text
 administration
-├── page
-│   └── sw-product-detail
+├── src
+│   └──page
```
Comment (sushmangupta) [inline guides/plugins/plugins/administration/add-new-tab.md:227]: @seggewiss , The actual path seems to be this `src/Administration/Resources/app/administration/src/module/sw-product/page/sw-product-detail/sw-product-detail.html.twig` . Should I include also include `modules/sw-product` after src directory?

File: guides/plugins/plugins/administration/add-new-tab.md [hunk 3]
```diff
@@ -223,13 +223,14 @@ As shown in the previous example, your custom component is expected to be in a d
...
 │       └── sw-product-detail
 │           ├── index.js
 │           └── sw-product-detail.html.twig
-├── view
-│   └── sw-product-detail-custom
+|── view
+│    └── sw-product-detail-custom
```
Comment (sushmangupta) [inline guides/plugins/plugins/administration/add-new-tab.md:232]: There is no directory named `sw-product-detail-custom`. Hope that's just an example name.

File: guides/plugins/plugins/administration/add-new-tab.md [hunk 4]
```diff
@@ -223,13 +223,14 @@ As shown in the previous example, your custom component is expected to be in a d
...
+│        ├── index.js
```
Comment (sushmangupta) [inline guides/plugins/plugins/administration/add-new-tab.md:233]: Added `index.js`, hope that's fine.

**Review by bojanrajh — APPROVED** [review]

**Pull request merged**